source/docs/appendices/migration-guides.md

```diff
@@ -61,6 +61,23 @@ In case you have overridden `_modules.scss` you need to add the following line t
 @import "~theme/modules/ProductView/ProductName/ProductName";
 ```
 
+### Cache Control and CDN
+
+To enable caching please:
+
+- Add `FRONT_COMMERCE_WEB_USE_GET_FOR_PERSISTED_QUERIES=true` environment variable (P.S. this will disable query batching)
+- Override `*CacheControlFragment.gql` fragments (Category / Product / CMS Page has the following fragments)
+- Also for static pages add a static attribute `cacheControlDefinition` with the following properties:
+
+```json5
+{
+  sMaxAge: 10,
+  staleWhileRevalidate: 1000,
+}
+```
+
+Please refer to [the Cache Control and CDN documentation](/docs/advanced/performance/cache-control-and-cdn) for a more in depth explanation about the topic
```

Comment:
this kind of duplicate the other page (with a less clear informations), I would
rather write something:

```
Front-Commerce 2.6 comes with a new powerful HTTP cache mechanism, Please refer to [the Cache Control and CDN documentation](/docs/advanced/performance/cache-control-and-cdn) for a more in depth explanation about the topic
```